mlhub/utils.py

```diff
@@ -2321,9 +2321,9 @@ def get_model_completion_list():
 def find_best_match(misspelled, candidates):
     """Find the best matched word with <misspelled> in <candidates>."""
 
-    best_match = fuzzprocess.extract(
+    best_match = fuzzprocess.extractOne(
         misspelled, candidates, scorer=fuzz.ratio
-    )[0]
+    )
```

Comment:
You are using is_misspelled on the result later on. It would be faster to directly use:
```
best_match = fuzzprocess.extractOne(
      misspelled, candidates, scorer=fuzz.ratio, score_cutoff=80
)
```
This returns None when there is no match above 80 and a tuple (choice, score) if there is one with a better score.